app/backend/approaches/chatreadretrieveread.py

```diff
@@ -170,9 +182,9 @@ async def run(self, history: Sequence[dict[str, str]], overrides: dict[str, Any]
             self.chatgpt_token_limit - len(user_question)
             )
 
-        chat_completion = await openai.ChatCompletion.acreate(
-            deployment_id=self.chatgpt_deployment,
-            model=self.model_name,
+        
+        chat_completion= await self.client.chat.completions.create(
+            model=self.chatgpt_deployment,
```

Comment:
are we sure this is correct? The deployment name may not match model name...

Reply:
This is correct parameter name. with new version model is deployment name. 
![image](https://github.com/microsoft/PubSec-Info-Assistant/assets/55892521/7827cf57-573d-43d5-be2b-8df4b2cc5c46)